exercises/practice/hello-world/.docs/instructions.append.md

```diff
@@ -66,73 +58,82 @@ After making corrections and implementing your solution, run the tests again.
+There are many more exercises you can practice with.
+Grab the next one! (example for `two-fer`)
 
 ```sh
 exercism download --exercise=two-fer --track=java
 ```
 
 ### Become a mentor
 
-The heart of Exercism are the conversations about coding
-practices. It's definitely fun to practice, but engaging with others
-both in their attempts and your own is how you get feedback. That feedback
-can help point out what you're doing well and where you might need to
-improve.
+The heart of Exercism are the conversations about coding practices.
+It's definitely fun to practice, but engaging with others both in their attempts and your own is how you get feedback.
+That feedback can help point out what you're doing well and where you might need to improve.
 
-Some submissions will be nearly identical to yours; others will be
-completely different. Seeing both kinds can be instructive and interesting.
+Some submissions will be nearly identical to yours, others will be completely different.
+Seeing both kinds can be instructive and interesting.
 
-Mentors review submitted solutions for the track they've chosen to mentor
-to help users learn as much as possible. To read more about mentoring and
-to sign up see: <http://mentoring.exercism.io/>.
+Mentors review submitted solutions for the track they've chosen to mentor to help users learn as much as possible.
 
-### Contribute to Exercism
-
-The entire of Exercism is Open Source and is a labor of love for over
-100 maintainers and many more contributors.
+To read more about mentoring, go to the following [link][Mentoring].
 
-A starting point to jumping in can be found here:
+### Contribute to Exercism
 
-<https://github.com/exercism/docs/tree/main/building>
+The entire of Exercism is Open Source and is a labor of love for over 100 maintainers and many more contributors.
+
+A starting point to jumping in and becoming a contributor can be found [here][Contributing].
+
+[Tutorial]: #tutorial
+[Introduction]: #introduction
+[Exercise Structure]: #exercise-structure
+[Solving "Hello, World!"]: #solving-hello-world
+[Step 1: Fix the solution]: #step-1-fix-the-solution
+[Step 2: Run the tests]: #step-2-run-the-tests
+[Step 3: Submitting your first iteration]: #step-3-submitting-your-first-iteration
+[Next steps]: #next-steps
+[Mentoring]: https://exercism.org/docs/mentoring
+[Contributing]: https://github.com/exercism/docs/tree/main/building
+[Installing Java]: https://exercism.org/docs/tracks/java/installation
+[Working Locally]: https://exercism.org/docs/using/solving-exercises/working-locally
+[Testing locally on the java track]: https://exercism.org/docs/tracks/java/tests
+[Testing on the Java track]: https://exercism.org/docs/tracks/java/tests
+[Exercism CLI]: https://exercism.org/docs/using/solving-exercises/working-locally
+[install the exercism CLI]: https://exercism.org/docs/using/solving-exercises/working-locally
+[Submitting locally]: https://exercism.org/docs/using/solving-exercises/working-locally
```

Comment:
Hmm I see at least 4 links pointing to the same URL here. Remember that you don't have to use the link text as the reference key, so you could do this too:

```md
Blah blah see [working locally][working-locally] blah blah.  
...  
Blah blah use the [Exercism CLI][working-locally] blah blah.  
...  
Blah blah [install the Exercism CLI][working-locally] blah blah.  
...  
Blah blah for that, check out [submitting locally][working-locally].

...

[working-locally]: https://exercism.org/docs/using/solving-exercises/working-locally
```

In this example all four links point to the same URL because they share the same reference, but they all have a different link text.

Reply:
You're right! going to fix it

Reply:
Done!